docs/UserGuide.md

```diff
@@ -115,9 +115,9 @@ Format: `list`
 
 ### Editing a person : `edit`
 
-Edits an existing client's parameter in FAPro.
+Edit a parameter in existing contacts, such as name, address, phone number, email, occupation, appointment date and tags.
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [o/OCCUPATION] [a/ADDRESS] [t/TAG]…​`
+Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [o/OCCUPATION] [a/ADDRESS] [appt/APPOINTMENTDATE] [t/TAG]…​`
```

Comment:
Maybe can change the "Format:" into "Precise command format:" as we all have agreed.

Reply:
done!